pkg/odo/cli/catalog/list/services.go

```diff
@@ -122,3 +123,22 @@ func NewCmdCatalogListServices(name, fullName string) *cobra.Command {
 		},
 	}
 }
+
+type catalogListOutput struct {
+	v1.TypeMeta   `json:",inline"`
+	v1.ObjectMeta `json:"metadata,omitempty"`
+	Services      *catalog.ServiceTypeList `json:"services,omitempty"`
+	// list of clusterserviceversions (installed by Operators)
+	Operators *olm.ClusterServiceVersionList `json:"operators,omitempty"`
+}
+
+func newCatalogListOutput(services *catalog.ServiceTypeList, operators *olm.ClusterServiceVersionList) catalogListOutput {
+	return catalogListOutput{
+		TypeMeta: v1.TypeMeta{
+			Kind:       "List",
+			APIVersion: machineoutput.APIVersion,
+		},
+		Services:  services,
+		Operators: operators,
+	}
+}
```

Comment:
I think we should move this to /pkg/catalog/catalog.go

Reply:
The only spot where this function and the associated type are used is this file so I don't think it makes sense to move it elsewhere for now.